src/core/org/apache/jmeter/resources/messages_fr.properties

```diff
@@ -853,7 +853,7 @@ reportgenerator_summary_statistics_kbytes=Ko/sec
 reportgenerator_summary_statistics_label=Libell\u00E9
 reportgenerator_summary_statistics_max=Max
 reportgenerator_summary_statistics_min=Min
-reportgenerator_summary_statistics_percentile_fmt=%d%% Ligne
+reportgenerator_summary_statistics_percentile_fmt=%de percentile
```

Comment:
Hi Milamber,

I have compare HTML report with other listener and find another difference (in % error)

I have commit it in https://github.com/apache/jmeter/pull/190 because percentil PR (PR 187) modify also a java file and not only properties files

Antonio